central_service/ansible/roles/rabbitmq/tasks/main.yml

```diff
@@ -1,5 +1,20 @@
 ---
-  - name: Include tasks for rabbitmq.
-    include: install.yml
-    tags:
+
+- name: Install RabbitMQ server
+  apt: name=rabbitmq-server install_recommends=no force=yes
+  tags:
+      - celery-install
+
+- name: Configure RabbitMQ user
+  rabbitmq_user: user=lambda password=change_me
```

Comment:
Should extract user and password for RabbitMQ user to the role's vars directory.